### PR TITLE
Bump up CDAP and Hadoop version to fix the log4j vulnerability

### PR DESCRIPTION
**Pull request by PandiyanA**
Bumped up the versions for CDAP and Hadoop to latest.

### CODE REVIEW

**Review by dli357 — DISMISSED** [review]

**Review by dli357 — COMMENTED** [review]
Oops didn't mean to approve; I think there was some offline discussion that this may require a  fix for the unit tests before we can bump the hadoop version.

File: pom.xml [hunk 1]
```diff
@@ -27,8 +27,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <cdap.version>6.1.1</cdap.version>
-        <hadoop.version>2.8.0</hadoop.version>
+        <cdap.version>6.8.0</cdap.version>
+        <hadoop.version>2.10.2</hadoop.version>
         <hydrator.version>2.3.0-SNAPSHOT</hydrator.version>
```
Comment (dli357) [inline pom.xml]: Please bump hydrator plugins version to 2.10.0 as it depends on CDAP. Otherwise LGTM, thanks!

**Comment by feefs-google** [comment]
Unit tests are failing because of dependency on `cdap-explore`. Since we're planning to remove it, let's leave this PR open until then.

**Comment by feefs-google** [comment]
After removing `cdap-explore` and bumping hydrator plugins to 2.10.2, unit tests that don't require credentials are passing, squashing and merging.

**Pull request merged**